Tools/CISupport/ews-app/ews/common/github.py

```diff
@@ -213,7 +213,7 @@ class GitHubEWS(GitHub):
                           ['jsc-arm64', 'vision', 'mac-AS-debug-wk2', 'gtk-wk2', ''],
                           ['services', 'vision-sim', 'mac-wk2-stress', 'api-gtk', ''],
                           ['merge', 'vision-wk2', 'mac-intel-wk2', 'jsc-armv7', ''],
-                          ['unsafe-merge', 'tv', '', 'jsc-armv7-tests', ''],
+                          ['unsafe-merge', 'tv', 'mac-safer-cpp', 'jsc-armv7-tests', ''],
```

Comment:
Would be a good idea to add the status-bubble in a separate PR which would land little later, once the queue is running properly in Production (without any issues like false positives). Meanwhile, make sure that this queue doesn't block/comment on the PRs.
Especially important for this queue, since this queue is expected to directly impact the engineers and they would have to learn some new ways of resolving issues found by ews queue.

However, if adequate testing has been done, it's fine to land all these changes together.

Reply:
This queue has been running on dev for a couple weeks and the results have been monitored and shared with the engineers working on the checker. I think we have enough testing that we can land all the changes together and comment on PRs.